test-pad/test-pad

```diff
@@ -0,0 +1,313 @@
+#!/bin/sh
+
+. "../bin/libstd-ecm.sh"
```

Comment:
```suggestion
. libstd-ecm.sh
```

This exists in the path in the image and is loadable from there.

Reply:
This tool will not be run via a Docker container. This is envisioned as a standalone tool,  otherwise we are launching a docker container to then launch VMs within it (not a road I want to go down).

Reply:
Since iti's not run within Docker, my comment can be ignored.